libs/state/docs/concepts-and-best-practices.md

```diff
@@ -0,0 +1,133 @@
+# General
+
+**Component Shell and Folder**
+
+- `@Input` bindings are setter
+- `@Output` bindings are state derivations
+- State is injected over the constructor
+- State is displayed over a pipe in the template
+- UI interaction is implemented over `Subjects`
+- use `*ngrxLet` over `*ngIf`
+  Bad:
+
+```html
+<ng-container *ngIf="obj$ | async as obj">
+  {{obj}}
+</ng-container>
+```
+
+Good:
+
+```html
+<ng-container *ngrxLet="obj$; let obj">
+  {{obj}}
+</ng-container>
+```
+
+**Logic:**
```

Comment:
this chapter doesn't make much sense right now?

Reply:
deleted it